.gitignore

```diff
@@ -1,2 +1,3 @@
 build/*
 index.json
+soupault
```

Comment:
This is specific to your setup, and will not be useful for people who install soupault system-wide, or somewhere in the `$PATH`.